tools/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.targets

```diff
@@ -75,13 +75,11 @@
     </OutputHashTask>
 
     <!-- Run build telemetry -->
-    <BuildTelemetryTask
+    <Exec
       WorkingDirectory="$(IceRpcBuildTelemetryScriptPath)"
-      CompilationHash="$(Hash)"
-      SourceFileCount="$(FileCount)"
+      Command="dotnet IceRpc.BuildTelemetry.Reporter.dll --idl protobuf --hash=$(Hash) --src-file-count=$(FileCount)"
       Condition="'$(IceRpcBuildTelemetry)' != 'false'"
-      ContinueOnError="true"
-    />
+      ContinueOnError="true" />
```

Comment:
Alternatively we can use `IgnoreExitCode` parameter from Exec task. The difference is that errors would be reported as regular messages instead of warnings.

Reply:
I think that `IgnoreExitCode` would be preferable.

Maybe our collective sentiment has changed, but I think that a telemetry failure should have no impact on whether your build succeeds or not; and emitting warnings would still cause a failure if warnings are being treated as errors (something that seems to be at least slightly common).

Reply:
Presumably ContinueOnError=true means we keep going on error.

Keeping in mind this process (slice compilation) is under our control here.

Reply:
Maybe I misunderstood, but #4049 made it sound like `ContinueOnError` was insufficient.
Even though, "keep going on error" is definitely the impression you get from it's name.